unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapToastView.cs

```diff
@@ -145,32 +149,30 @@ private void OnGotoClick()
         }
 
         string currentImageUrl;
-        Texture currentImage;
+        Texture2D currentImage;
         Coroutine downloadCoroutine;
 
         private IEnumerator Download(string url)
         {
+            scenePreviewLoadingSpinner.SetActive(true);
+            scenePreviewImage.texture = null;
+
             UnityWebRequest www = UnityWebRequestTexture.GetTexture(url);
```

Comment:
Maybe not in the scope of this PR, but for the record, this should be using `AssetPromiseKeeper_Texture`.

Reply:
Yeap, I was going to add that after this PR cause we are in a hurry to free those prefabs